.github/workflows/cmake-linux-ubuntu.yml

```diff
@@ -43,4 +43,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: web-eid-app-ubuntu-build-${{github.run_number}}
-          path: build/src/app/*.deb
+          path: build/src/app/*.*deb
```

Comment:
Maybe `*.?(d)deb` or `*.{deb,ddeb}` would be more clear? See glob spec [here](https://github.com/isaacs/node-glob/#glob-primer) and test tool [here](https://www.digitalocean.com/community/tools/glob).

Reply:
Let's use `*.{deb,ddeb}` for now and add other artifacts as required (by YAGNI).